### PR TITLE
docs: update deploy-with-docker.md

### PR DESCRIPTION
**Pull request by FangSen9000**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

apisix-dashboard login success,then access apisix/admin/labels/route or apisix/admin/labels/consumer ，encounter to get 401 Unauthorized error code,which means the jwt token that authoried by dashboard is invalid.

finally with community's help，found that the reason is more than 1 dashboard instance.
with multi instance, every instance generate and hold a jwt token,will cause validation conflicted

so we can optimize the doc to emphasize the dashboard instance number not more than 1,DIFFERENT from high availability with apisix itself.

optimize the doc,emphasize the dashboard instance number not more than 1

I have added comments in the appropriate place in the Docker-related documentation.

**Related issues**

fix/resolve [doc] 仪表板 Docker 实现使用优化 issues[#2446](https://github.com/apache/apisix-dashboard/issues/2446)

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by FangSen9000** [comment]
@Baoyuantop, I accidentally changed the file name before, but only changed the content this time.

File: docs/en/latest/deploy-with-docker.md [hunk 1]
```diff
@@ -87,3 +87,5 @@ $ docker stop apisix-dashboard
 ```sh
 $ docker build -t apisix-dashboard:$tag . --no-cache=true
 ```
+
+2.  It is not recommended to use multiple instances at the same time.  When using multiple instances, each instance generates and holds a JWT token, which will lead to verification conflicts.
```
Comment (SkyeYoung) [inline docs/en/latest/deploy-with-docker.md:91]:
```suggestion
2.  It is not recommended to use multiple instances at the same time. When using multiple instances, each instance generates and holds a JWT token, which will lead to verification conflicts.
```

**Review by SkyeYoung — CHANGES_REQUESTED** [review]
Pls refer to [other PRs](https://github.com/apache/apisix-dashboard/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) to edit your title.

**Comment by SkyeYoung** [comment]
Pls also revise the description of the PR in English to make it accurate and clear.

**Review by kafkahz — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**